compiler/tflchef/core/src/OpUtils.cpp

```diff
@@ -20,9 +20,8 @@
 
 void check_custom_op_value(const tflchef::Operation operation, std::string op_type)
 {
-  if ((operation.has_extype() && operation.extype() == "Custom") || operation.type() == "Custom")
+  if (operation.type() == "Custom" || (operation.has_extype() && operation.extype() == "Custom"))
```

Comment:
let's change this in another PR. I don't think this change is related with purpose of this PR.